package.json

```diff
@@ -23,12 +23,6 @@
   "engines": {
     "node": ">=0.10.0"
   },
-  "files": [
-    "app",
-    "dispatcher",
-    "store",
-    "action"
-  ],
```

Comment:
Was this causing issues as well? Was trying to follow the [spec](https://docs.npmjs.com/files/package.json).